bifrost/app/blog/blogs/best-langsmith-alternatives/src.mdx

```diff
@@ -317,4 +318,44 @@ OpenLLMetry aims to standardize the collection of mission-critical LLM metrics,
 
 Traceloop focuses on evaluation and the pricing reflects that, thus can become expensive as your application scale to log more traces. 
 
-#### So, which LangSmith alternatives suits you better?
\ No newline at end of file
+---
+
+## Frequently Asked Questions
+
+### Q: Which LangSmith alternative is best for open-source and self-hosting?
+
+Helicone and Langfuse are both open-source and offer self-hosting options, giving you full control over your data and infrastructure.
+
+### Q: How do these tools handle data privacy and security?
+
+- **Helicone**: Being open-source and self-hostable, Helicone allows you to keep all data within your infrastructure, enhancing security.
```

Comment:
Also SOC2 compliance. Privacy terms

Reply:
would also love some wording suggestions here

Reply:
Link to our privacy policy https://us.helicone.ai/privacy, mentioning SOC2 compliance, ability to omit log bodies from longer-term storage, and then for more extreme measures, open-source and scalable self-hosting capabilities